textnow_bot.py

```diff
@@ -1,4 +1,6 @@
 import logging
+import inspect
+import asyncio
```

Comment:
Remove unused `asyncio` import, and also reorder imports for linters.

```suggestion
import inspect
import logging
```